src/test/java/sk/freemap/gpxAnimator/ui/DurationFormatterTest.java

```diff
@@ -0,0 +1,42 @@
+package sk.freemap.gpxAnimator.ui;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.text.ParseException;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class DurationFormatterTest {
+
+    private static Stream<Arguments> generateGetterSetterTestData() {
```

Comment:
The mthod name is not very good in this case. In this class it does not generate data to test the getter or setter. Will fix this.